xbmc/video/VideoDatabase.cpp

```diff
@@ -1025,6 +1025,34 @@ bool CVideoDatabase::GetLinksToTvShow(int idMovie, std::vector<int>& ids)
   return false;
 }
 
+bool CVideoDatabase::GetLinksToTvShow(std::map<int, CVideoInfoTag>& details)
```

Comment:
I'm not a fan of the way the parameters are passed in. The method expects a half-filled map where the first value of every pair is already properly initialized but the second value isn't. IMO that's not obvious at all.
I also prefer it if output parameters aren't also used as input parameters.